Documentation/Books/Cookbook/Compiling/Windows.md

```diff
@@ -4,20 +4,20 @@ Compiling ArangoDB under Windows
 Problem
 -------
 
-I want to compile ArangoDB 3.0 and onwards under Windows.
+I want to compile ArangoDB 3.4 and onwards under Windows.
 
-**Note:** For this recipe you need at least ArangoDB 3.0;
-For ArangoDB version before 3.0 look at the [old Compiling ArangoDB under Windows](https://docs.arangodb.com/2.8/Cookbook/CompilingUnderWindows.html).
+**Note:** For this recipe you need at least ArangoDB 3.4;
+For ArangoDB version before 3.4 look at the [old Compiling ArangoDB under Windows](https://docs.arangodb.com/3.0/Cookbook/CompilingUnderWindows.html).
 
 Solution
 --------
 
-With ArangoDB 3.0 a complete cmake environment was introduced. This also streamlines the dependencies on windows.
+With ArangoDB 3.0 a complete cmake environment was introduced. This also streamlines the dependencies on Windows.
 We suggest to use [chocolatey.org](https://chocolatey.org/) to install most of the dependencies. For sure
 most projects offer their own setup & install packages, chocolatey offers a simplified way to install them
 with less user interactions. You can even use chocolatey via the brand new
 [ansibles 2.0 winrm facility](http://docs.ansible.com/ansible/intro_windows.html)
-to do unattended installations of some software on windows - the cool thing linux guys always told you about.
+to do unattended installations of some software on Windows - the cool thing Linux guys always told you about.
```

Comment:
"the cool thing Linux guys always told you about." - probably we should remove this comment